cmd/nerdctl/stats_linux_test.go

```diff
@@ -32,9 +33,13 @@ func TestStats(t *testing.T) {
 		t.Skip("test skipped for rootless containers on cgroup v1")
 	}
 	testContainerName := testutil.Identifier(t)
+	exitedTestContainerName := fmt.Sprintf("%s-exited", testContainerName)
 
 	base := testutil.NewBase(t)
 	defer base.Cmd("rm", "-f", testContainerName).Run()
+	defer base.Cmd("rm", "-f", exitedTestContainerName).Run()
+	base.Cmd("run", "--name", exitedTestContainerName, testutil.AlpineImage, "echo", "'exited'").AssertOK()
+	base.Cmd("kill", exitedTestContainerName).AssertOK()
```

Comment:
no need to kill this container is already exited https://github.com/containerd/nerdctl/pull/1416/files#diff-c1ea01872000c1855f6cfc8180843ee41c96d0f5d54cc479681a96956caf6a58R41